### DISCUSSION TITLE
Trilium, custom domain name, Digital Ocean and Docker

### DISCUSSION THREAD

**Discussion by sottey** [TriliumNext/Trilium#4170] [Q&A] [answered]
All,

Sorry in advance but I am a networking dummy.

I have a domain which is set up for https. I have an account at Name.com with a Digital Ocean cloud hosting plan. In that droplet I am running Trilium using the docker image.

I don't even know where to start to try and get https to work. When I go to the domain, if I use http, it works great. If I go via https, it says:
<img width="439" alt="image" src="https://github.com/zadam/trilium/assets/8494620/1b302b88-171c-4999-b2ad-9da55cf0a68d">

I am not asking for the answer, necessarily, but could someone point me in the right direction as to where I should start?

Thanks!
Sean

**Comment by Nriver** [comment, marked as answer]
You need to setup a reverse proxy server like Nginx. https://github.com/zadam/trilium/wiki/Nginx-proxy-setup
